.buildkite/scripts/steps/es_snapshots/build.sh

```diff
@@ -79,11 +79,25 @@ find distribution -type f \( -name 'elasticsearch-*-*-*-*.tar.gz' -o -name 'elas
 
 ls -alh "$destination"
 
-echo "--- Create docker image archives"
+echo "--- Create docker default image archives"
 docker images "docker.elastic.co/elasticsearch/elasticsearch"
 docker images "docker.elastic.co/elasticsearch/elasticsearch" --format "{{.Tag}}" | xargs -n1 echo 'docker save docker.elastic.co/elasticsearch/elasticsearch:${0} | gzip > ../es-build/elasticsearch-${0}-docker-image.tar.gz'
 docker images "docker.elastic.co/elasticsearch/elasticsearch" --format "{{.Tag}}" | xargs -n1 bash -c 'docker save docker.elastic.co/elasticsearch/elasticsearch:${0} | gzip > ../es-build/elasticsearch-${0}-docker-image.tar.gz'
 
+echo "--- Create kibana-ci docker cloud image archives"
+ES_CLOUD_ID=$(docker images "docker.elastic.co/elasticsearch-ci/elasticsearch-cloud" --format "{{.ID}}")
+ES_CLOUD_VERSION=$(docker images "docker.elastic.co/elasticsearch-ci/elasticsearch-cloud" --format "{{.Tag}}")
+KIBANA_ES_CLOUD_VERSION="$ES_CLOUD_VERSION-$ELASTICSEARCH_GIT_COMMIT"
+KIBANA_ES_CLOUD_TAG="docker.elastic.co/kibana-ci/elasticsearch-cloud:$KIBANA_ES_CLOUD_VERSION"
+
+docker tag "$ES_CLOUD_ID" "$KIBANA_ES_CLOUD_TAG"
+echo "docker save docker.elastic.co/kibana-ci/elasticsearch-cloud:$KIBANA_ES_CLOUD_VERSION | gzip > ../es-build/elasticsearch-cloud-$ES_CLOUD_VERSION-docker-image.tar.gz"
+docker save "docker.elastic.co/kibana-ci/elasticsearch-cloud:$KIBANA_ES_CLOUD_VERSION"  | gzip > ../es-build/elasticsearch-cloud-$ES_CLOUD_VERSION-docker-image.tar.gz
```

Comment:
What is this archive used for?

Reply:
In practice probably nothing - I added it as a signal of it's existence in manifest.json.  I'm okay with removing it.

Reply:
Maybe just put the docker url in the manifest then? Seems like probably a large file to generate and upload if it's not necessary. I'm good either way

Reply:
I pushed an update for this in https://github.com/elastic/kibana/pull/127154/commits/b6434d48e70d562a43fe05cd257a4005a1cb9b86